src/AutoComplete/style.css

```diff
@@ -0,0 +1,22 @@
+@import '../assets/style/form-item.css';
```

Comment:
only related to the form. Should be imported in the story

Reply:
Are they styles below also depending on a form? if yes please remove this dependency

Reply:
auto complete default style import missing